assets/less/content/general.less

```diff
@@ -53,7 +53,7 @@ a.no-underline {
 
 a.view-source {
   float: right;
-  color: @mediumGray;
+  color: #727272;
```

Comment:
Should we be replacing variable usages by hard-coded entries? Maybe we should make `@mediumGray`itself be #727272?

Reply:
This was an edge case. In a regular situation that would be fine, but because there is a different background this color needs to be changed. If we replace @mediumGray it will look darker that in was originally designed for the rest of the pages. I did not want to change much the design. That' s why.